tests/ops/integration_tests/saas/connector_runner.py

```diff
@@ -142,11 +143,13 @@ async def access_request(
             identities,
             self.db,
         )
-        # verify we returned at least one row for each collection in the dataset
-        for collection in self.dataset["collections"]:
-            assert len(
-                access_results[f"{fides_key}:{collection['name']}"]
-            ), f"No rows returned for collection '{collection['name']}'"
+
+        if not skip_collection_verification:
```

Comment:
Small change to our connector runner to skip collection verification. In our case, the user collection will be empty but that should be allowed